examples/js/plugins/FeatureToolTip.js

```diff
@@ -39,7 +39,7 @@ var FeatureToolTip = (function _() {
         tooltip.innerHTML = '';
         tooltip.style.display = 'none';
 
-        var features = view.pickFeaturesAt(event, 3, layersId);
+        var features = view.pickFeaturesAt.apply(view, [event, 3].concat(layersId));
```

Comment:
Why you use `pickFeaturesAt`?

Reply:
Because of the `...` operator, that we don't allow in plugins (we allow only ES5 syntax). See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#Replace_apply

Reply:
ah, yes!